rosdep/python.yaml

```diff
@@ -2672,6 +2672,25 @@ python-pytest:
   fedora: [python-pytest]
   gentoo: [dev-python/pytest]
   ubuntu: [python-pytest]
+python-pytest-qt:
+  debian:
+    buster: [pytest-qt]
```

Comment:
there is no binary deb available on buster yet.

Reply:
You are right. There is one for python3 though: https://packages.debian.org/buster/python3-pytestqt

Reply:
> There is one for python3 though: https://packages.debian.org/buster/python3-pytestqt

:+1:
Currently there is no ROS distribution targeting Python3. Though that will likely change in the future https://github.com/ros-infrastructure/rep/pull/149